ionic_user_interface/src/views/Signup.vue

```diff
@@ -70,6 +70,19 @@
                 >
                   Sign up
                 </ion-button>
+                <div class="privacy-policy">
+                  <ion-checkbox v-model="termsCheckbox" />
+                  <ion-text>
```

Comment:
![image](https://user-images.githubusercontent.com/46164638/126195022-5fcc50ee-310d-4708-bda1-c4cbe1de311c.png)

Need to import.